### ISSUE TITLE
keras autlog not working with Model API

### ISSUE THREAD

**Issue by hkropp** [mlflow/mlflow#2336]
The following exception is thrown when using `mflow.keras.autolog()` with Keras Model API.

```
/databricks/python/lib/python3.7/site-packages/mlflow/keras.py:432: UserWarning: Logging to MLflow failed: INVALID_PARAMETER_VALUE: Tag Model: "model_1"
```

The model is instantiated with:
```
model = Model(inputs = X_input, outputs = Xfc2)
```

**Comment by tomasatdatabricks** [comment]
Hi @hkropp, thanks for posting!

Are you able to save the same model via mlflow.keras.save_model?

**Comment by smurching** [comment]
@hkropp apologies for the delay - to confirm, this issue causes a warning to be logged but doesn't result in failure of your code, right? The warning should go away once MLflow 1.8.0, which contains https://github.com/mlflow/mlflow/pull/2703, is released

**Comment by smurching** [comment]
Should be fixed in MLflow 1.8.0
